python/src/scipp/table_html/formatting_html.py

```diff
@@ -0,0 +1,396 @@
+    }
+
+    dims_li = "".join(
+        f"<li><span{dim_css_map[dim]}>" f"{escape(str(dim))}</span>: "
+        f"{size if size != sc.Dimensions.Sparse else 'Sparse' }</li>"
+        for dim, size in zip(dims, sizes)
+    )
+
+    return f"<ul class='xr-dim-list'>{dims_li}</ul>"
+
+
+def summarize_attrs_simple(attrs):
+    attrs_dl = "".join(
+        f"<dt><span>{escape(name)} :</span></dt>" f"<dd>{values}</dd>"
+        for name, values in attrs
+    )
+
+    return f"<dl class='xr-attrs'>{attrs_dl}</dl>"
+
+
+def summarize_attrs(attrs):
+    attrs_li = "".join(
+        f"<li class='xr-var-item'>\
+            {summarize_variable(name, values, is_attr=True)}</li>"
+        for name, values in attrs
+    )
+    return f"<ul class='xr-var-list'>{attrs_li}</ul>"
+
+
+def _icon(icon_name):
+    # icon_name is defined in icon-svg-inline.html
+    return (
+        "<svg class='icon xr-{0}'>"
+        "<use xlink:href='#{0}'>"
+        "</use>"
+        "</svg>".format(icon_name)
+
+    )
+
+
+def summarize_coord(dim, var):
+    is_index = dim in var.dims
+    return summarize_variable(str(dim), var, is_index)
+
+
+def summarize_coords(coords):
+    vars_li = "".join(
+        f"<li class='xr-var-item'>{summarize_coord(dim, values)}</span></li>"
+        for dim, values in coords)
+
+    return f"<ul class='xr-var-list'>{vars_li}</ul>"
+
+
+def summarize_variable(name, var, is_index=False, is_attr=None):
+    """
+    :param name:
+    :param var:
+    :param is_index: If the variable is an index - used to bold
+                     coordinates that represent the indices of
+                     a dimension that the data contains
+
+    :param is_attr: If the variable is for an attribute, then this
+                    hides the show/hide attributes button.
+
+    """
+    cssclass_idx = " class='xr-has-index'" if is_index else ""
+    dims_text = ', '.join(escape(f'{str(dim)} [sparse]'
+                                 if dim == var.sparse_dim
+                                 else str(dim))
+                          for dim in var.dims)
+    dims_str = f"({dims_text})"
+    name = escape(name)
+    dtype = var.dtype
+    unit = '' if var.unit == sc.units.dimensionless else var.unit
+
+    # "unique" ids required to expand/collapse subsections
+    attrs_id = "attrs-" + str(uuid.uuid4())
+    data_id = "data-" + str(uuid.uuid4())
+    if hasattr(var, "attrs"):
+        disabled = "" if len(var.attrs) > 0 else "disabled"
+        attrs_ul = summarize_attrs_simple(var.attrs)
+    else:
+        disabled = "disabled"
+        attrs_ul = None
+
+    preview = inline_variable_repr(var)
+    data_repr = f"Values:<br>{short_data_repr_html(var)}"
+    variances_preview = ""
+    if var.variances is not None:
+        variances_preview = inline_variable_repr(var, has_variances=True)
+        data_repr += f"<br><br>Variances:<br>\
+                       <div>{short_data_repr_html(var)}</div>"
```

Comment:
Somehow there is an extra empty line after "Variances:" which is not there after "Values:". Is it due to the `<div>`?

Reply:
I cannot see it, could you post a screenshot if possible?